pkg/oc/cli/cmd/startbuild.go

```diff
@@ -98,6 +98,7 @@ func NewCmdStartBuild(fullName string, f *clientcmd.Factory, in io.Reader, out,
 		SuggestFor: []string{"build", "builds"},
 		Run: func(cmd *cobra.Command, args []string) {
 			kcmdutil.CheckErr(o.Complete(f, in, out, errout, cmd, fullName, args))
```

Comment:
While you're at it, please move `in, out, errout` to line 91 where `StartBuildOptions` is instantiated.